src/alfasim_sdk/_internal/alfacase/alfacase.py

```diff
@@ -26,14 +26,15 @@ def _generate_alfatable_file_for_pvt_models_description(
     """
     from alfasim_sdk import generate_alfatable_file
 
-    for pvt_name, pvt_table_description in pvt_models.table_parameters.items():
+    # TODO: ASIM-4887: generate alfatable files for ph table
```

Comment:
Note that this `TODO` points to the same issue your used for this branch. Do you intend to close the issue right after this merge? If so, this TODO task id should be updated to another issue. If no, nevermind 😄